fawkes/utils/utils.py

```diff
@@ -131,3 +132,23 @@ def fetch_channel_config(app_config, channel_type):
             return review_channel
     return None
 
+def write_query_results(response, write_file, format):
+    # Create the directory structure if does not exists else pass
+    try:
+        # Split the response file path to get directory
+        query_response_directory_path = os.path.split(write_file)
+        Path(query_response_directory_path[0]).mkdir(parents=True, exist_ok=True)
+    except FileExistsError:
+        pass
+    if format == constants.JSON:
+        with open(write_file, "w") as file:
```

Comment:
There's already a function `dump_json` in the same file. Try to re-use that.

Also this is an ideal time to create a function `dump_csv` and re-use that here